static/js/sefaria/sefaria.js

```diff
@@ -1926,7 +1932,7 @@ _media: {},
     // Used when isACaseVariant() is true to prepare the alternative
     return data["completions"][0] + query.slice(data["completions"][0].length);
   },
-  makeSegments: function(data, withContext) {
+  makeSegments: function(data, withContext, sheets=false) {
```

Comment:
I would not strip images in this function since it adds unnecessary responsibility to it. Instead, let's create another function that strips images from segments by looping over the segments created from `makeSegments()`. That should encapsulate the image stripping logic and make it easy to modify once we have more clarity on how images should display.